CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### citus v12.1.6 (Nov 14, 2024) ###
+
+* Propagates `SECURITY LABEL .. ON ROLE` statements (#7304)
```

Comment:
It is a weird change for a patch release. However, I am aware that this is not the correct place to discuss this here.